docs/integrations/sources/mongodb.md

```diff
@@ -0,0 +1,67 @@
+# Mongodb 
+
+The MongoDB source supports Full Refresh only.
+
+## Resulting schema 
+MongoDB does not have anything like table definition, thus we have to define column types from actual attributes and their values. Discover phase have two steps:
+
+### Step 1. Find all unique properties
+Connector runs the map-reduce command which returns all unique document props in the collection. Map-reduce approach should be sufficient even for large clusters.
+
+### Step 2. Determine property types
+For each property found, connector selects 10k documents from the collection where this property is not empty. If all the selected values have the same type - connector will set appropriate type to the property. In all other cases connector will fallback to `string` type.
+
+## Features
+
+| Feature | Supported |
+| :--- | :--- |
+| Full Refresh Sync | Yes |
+| Incremental - Append Sync | No |
+| Replicate Incremental Deletes | No |
+
+## Getting started
+Usually MongoDB is used locally behind the firewall without authentication. This guide describes in details how you can configure MongoDB for integration with Airbyte.
+
+### Create users
+
+Run `mongo` shell, switch to `admin` database and create three users: 
+
+1. ADMIN_USER will be used for database administration (you can skip this user in case you already use authentication for MongoDB access).
```

Comment:
I think Airbyte's instructions should only describe the read only user case because presumably someone syncing data from Mongo already has a mongo instance setup, so we shouldn't give instructions about administrating the DB or interacting with it from their application. The scope of this guide is larger/requires more maintenance if we give instructions for maintaining the Mongo database instance. WDYT?

Reply:
That's reasonable. I've narrowed the scope of instructions. Please let me know if you think I should drop even more information.